From 292030093e6a3861d731f632e89de2536e428441 Mon Sep 17 00:00:00 2001
From: allen yan <yanwei@marvell.com>
Date: Wed, 19 Apr 2017 17:22:13 +0800
Subject: [PATCH 0718/1200] arm64: dts: a3700: update devel-board dts model and
 compatible string

- This patch updates devel-board dts file's model and compatible string
  to make it work for Armada 3720 DDR3 DB v2 board:
  - "v2" implies devel-board's reversion 2.x.
  - "DDR3" implies the type of ddr device mounted on the board.
    document 'armada-3700-db-setup.txt' for details.
- More serdes/board configuration will be supported with later
  patches.

Change-Id: I0c4566a20950630a2c1ec9c0cda6dc4fcdd6cc4e
Signed-off-by: allen yan <yanwei@marvell.com>
Signed-off-by: Ken Ma <make@marvell.com>
Reviewed-on: http://vgitil04.il.marvell.com:8080/54450
Reviewed-by: Igal Liberman <igall@marvell.com>
Tested-by: Igal Liberman <igall@marvell.com>
---
 arch/arm/dts/armada-3720-db.dts | 6 ++++--
 1 file changed, 4 insertions(+), 2 deletions(-)

diff --git a/arch/arm/dts/armada-3720-db.dts b/arch/arm/dts/armada-3720-db.dts
index 2293b692b9..135f1f21ea 100644
--- a/arch/arm/dts/armada-3720-db.dts
+++ b/arch/arm/dts/armada-3720-db.dts
@@ -49,8 +49,10 @@
 #include "armada-372x.dtsi"
 
 / {
-	model = "Marvell Armada 3720 Development Board DB-88F3720-DDR3";
-	compatible = "marvell,armada-3720-db", "marvell,armada3720", "marvell,armada3710";
+	model = "Marvell Armada 3720 Development Board DB-88F3720-DDR3-V2";
+	compatible = "marvell,armada-3720-db-v2",
+		     "marvell,armada3720",
+		     "marvell,armada3710";
 
 	chosen {
 		stdout-path = "serial0:115200n8";
-- 
2.22.0

